core/config.py

```
@@ -0,0 +1,16 @@
+
+class Settings(BaseSettings):
+    MONGO_URI: str
+    MONGO_DB: str = "auth_db"
+    JWT_SECRET: str
+    JWT_ALGORITHM: str = "HS256"
+    ACCESS_TOKEN_EXPIRE: int = 30  # minutes
+
+    model_config = SettingsConfigDict(env_file=".env", extra='ignore')
+
+
+@lru_cache
```

Ruff (0.8.2)

Undefined name `lru_cache`

(F821)